packages/nimble-components/src/label-provider/table/index.ts

```diff
@@ -89,6 +93,12 @@ export class LabelProviderTable
     @attr({ attribute: 'row-loading' })
     public rowLoading: string | undefined;
 
+    @attr({ attribute: 'group-row-placeholder-no-value' })
```

Comment:
I learned that there is work to do in Angular and Blazor for new label provider strings too. See #1927. It probably makes sense to do in this PR but would also be fine in a follow up.

Reply:
I was planning on doing it in a different PR, but there isn't a reason not to include it here. I'll go ahead and do that.